config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,16 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+#: --tmpfs parameter
+tmpfs_path = /run
+run_options_csv = --tmpfs %(tmpfs_path)s
+#: command to run in container to check if passed --tmpfs is mounted as tmpfs type
+fndmnt = findmnt -n -t tmpfs %(tmpfs_path)s
+cmd = 'sleep 2s && %(fndmnt)s'
+run_append_name = True
```

Comment:
This is already the default, and it isn't especially germane to this test. Why not just remove it?